packages/example-react-app/src/WalletRpcPluginTest.tsx

```diff
@@ -0,0 +1,27 @@
+import { useContext } from "react";
+import { IWeb3Context, Web3Context } from "./web3/Web3Context";
+
+function WalletRpcPluginTest() {
+  const web3Context: IWeb3Context = useContext(Web3Context);
+
+  return (
+    <div>
+      <button
+        onClick={() =>
+          web3Context.web3.walletRpc.switchEthereumChain({ chainId: 137 })
```

Comment:
when trying to switch to this chainId i get an error 
<img width="1725" alt="Screenshot 2024-10-23 at 12 55 49 PM" src="https://github.com/user-attachments/assets/69d299cf-3fa2-409c-96b6-5a2f424ee5aa">

Reply:
yes, I still need to do a proper examples and testing in a real browser for all methods, that's coming in the next PR 🙂